tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Controllers/UsersControllerTests.cs

```diff
@@ -171,5 +171,51 @@ private static (ApplicationUser RepoUser, OrganisationUserViewModel ExpectedUser
                 }
             );
         }
+
+        [Test]
+        public async Task GetUserById_WithExistingUserId_ReturnsOkayObjectResult()
```

Comment:
We're testing more than just the result type here so probably best tweaking the name.